pkg/services/navtree/navtreeimpl/applinks.go

```diff
@@ -255,8 +255,9 @@ func (s *ServiceImpl) hasAccessToInclude(c *contextmodel.ReqContext, pluginID st
 func (s *ServiceImpl) readNavigationSettings() {
 	s.navigationAppConfig = map[string]NavigationAppConfig{
 		"grafana-k8s-app":                  {SectionID: navtree.NavIDMonitoring, SortWeight: 1, Text: "Kubernetes"},
-		"grafana-synthetic-monitoring-app": {SectionID: navtree.NavIDMonitoring, SortWeight: 2, Text: "Synthetics"},
+		"grafana-app-observability-app":    {SectionID: navtree.NavIDMonitoring, SortWeight: 2, Text: "Application (preview)"},
 		"grafana-kowalski-app":             {SectionID: navtree.NavIDMonitoring, SortWeight: 3, Text: "Frontend"},
```

Comment:
@cedricziel sorry for the late notice, should we also indicate "preview" for Frontend Observability?

Reply:
@cyrille-leclerc My assumption has been that App-o11y is preview, but Frontend Observability isn't; they're at different stages of development, so their states aren't directly correlated; correct me if I'm wrong @cedricziel

Reply:
Gonna merge this, let's figure out Frontend Observability separately :)

Reply:
Thanks, there is uncertainty on the GA date for Frontend O11y due to the pricing question. @cedricziel what would be your recommendation?